felix/bpf-gpl/bpf.h

```diff
@@ -226,17 +226,31 @@ static CALI_BPF_INLINE __attribute__((noreturn)) void bpf_exit(int rc) {
 
 #ifdef IPVER6
 
+#ifdef BPF_CORE_SUPPORTED
+#define IPv "[%pI6]"
```

Comment:
Could we make this more clearly related to formatting?  IP_FMT, maybe?

Reply:
yeah, I just did not have a better name, thanks!